icm-contracts/avalanche/teleporter/TeleporterMessenger.sol

```diff
@@ -108,6 +112,16 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
             => mapping(address feeTokenContract => uint256 redeemableRewardAmount)
     ) internal _relayerRewardAmounts;
 
+    /*
+     * @notice Register the address of the contract to be used for sending/receiving
+     * Warp messages.
+     * @dev This function is a delayed constructor to allow using Nick's method
+     * to deploy this contract
+     */
+    function initialize(address warpContract) external initializer {
```

Comment:
```
     * @dev This function is a delayed constructor to allow using Nick's method
     * to deploy this contract
```

I think this comment/rationale is incorrect. Nick's method txs can still provide constructor arguments for contract deployments.